src/save.ts

```diff
@@ -49,7 +49,7 @@ async function run(): Promise<void> {
                 uploadChunkSize: utils.getInputAsInt(Inputs.UploadChunkSize)
             });
             core.info(`Cache saved with key: ${primaryKey}`);
-        } catch (error) {
+        } catch (error: any) {
```

Comment:
Using `any` type should be avoided and `unknown` should be used here and everywhere else instead. That will required adding types here. Will work with @t-dedah  on doing this.

Reply:
Done